Conceptpower+Spring/src/main/java/edu/asu/conceptpower/root/DatabaseManager.java

```diff
@@ -7,44 +7,49 @@
 import com.db4o.cs.Db4oClientServer;
 import com.db4o.cs.config.ServerConfiguration;
 
+import edu.asu.conceptpower.servlet.core.ConceptEntry;
+
 public class DatabaseManager implements Serializable {
 
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = -3325272288078647257L;
-	private ObjectServer server;
-	private String databasePath;
-
-	public void init() {
-		close();
-		ServerConfiguration configuration = Db4oClientServer
-				.newServerConfiguration();
-		configuration.file().blockSize(80);
-		server = Db4oClientServer.openServer(configuration, databasePath, 0);
-	}
-
-	public ObjectContainer getClient() {
-		ObjectContainer container = server.openClient();
-		return container;
-	}
-
-	public String getDatabasePath() {
-		return databasePath;
-	}
-
-	public void setDatabasePath(String databasePath) {
-		this.databasePath = databasePath;
-	}
-
-	private void close() {
-		if (server != null) {
-			server.close();
-		}
-		server = null;
-	}
-
-	public void shutdown() {
-		close();
-	}
+    /**
+     * 
+     */
+    private static final long serialVersionUID = -3325272288078647257L;
+    private ObjectServer server;
+    private String databasePath;
+
+    public void init() {
+        close();
+        ServerConfiguration configuration = Db4oClientServer
+                .newServerConfiguration();
+        configuration.file().blockSize(80);
+        //Added to make sure list has been added to the database
+        //Ref: http://www.resolvinghere.com/sof/12343387.shtml
+        configuration.common().objectClass(ConceptEntry.class).updateDepth(2);
+        server = Db4oClientServer.openServer(configuration, databasePath, 0);
+    }
+
+    public ObjectContainer getClient() {
+        ObjectContainer container = server.openClient();
+        return container;
+    }
+
+    public String getDatabasePath() {
+        return databasePath;
+    }
+
+    public void setDatabasePath(String databasePath) {
+        this.databasePath = databasePath;
+    }
+
+    private void close() {
+        if (server != null) {
+            server.close();
+        }
+        server = null;
+    }
+
+    public void shutdown() {
+        close();
+    }
```

Comment:
@jdamerow I added these lines in init method

//Added to make sure list has been added to the database
        //Ref: http://www.resolvinghere.com/sof/12343387.shtml
        configuration.common().objectClass(ConceptEntry.class).updateDepth(2);

This line is added to make sure ChangeEvents list is getting stored in DB